cmd/pcap.go

```diff
@@ -35,21 +38,43 @@ func (c PcapCmd) Run(opts PcapOpts) error {
 		return err
 	}
 
-	result, err := c.deployment.SetUpSSH(opts.Args.Slug, sshOpts)
-	if err != nil {
-		return err
+	var result boshdir.SSHResult
+
+	slugs := []boshdir.AllOrInstanceGroupOrInstanceSlug{{}}
```

Comment:
I'm assuming a single empty Slug represents all instances by default? Please add a comment for that as you would usually initialise this differently.